tests/contenttypes_tests/test_checks.py

```diff
@@ -0,0 +1,247 @@
+from unittest import mock
+
+from django.contrib.contenttypes.fields import (
+    GenericForeignKey, GenericRelation,
+)
+from django.contrib.contenttypes.models import ContentType
+from django.core import checks
+from django.db import models
+from django.test import SimpleTestCase, override_settings
+from django.test.utils import isolate_apps
+
+
+@override_settings(SILENCED_SYSTEM_CHECKS=['fields.W342'])  # ForeignKey(unique=True)
```

Comment:
I think this is not required anymore now that these tests rely on `isolate_apps`?